plugins/outputs/inlong/sample.conf

```diff
@@ -0,0 +1,32 @@
+# Send telegraf metrics to Inlong
+[[outputs.inlong]]
+  ## From the inlong system, data Streams Group, it contains multiple data streams, and one Group represents
+  ## one data business unit.
+  inlong_group_id = "test_group"
```

Comment:
Could you comment out any config options here that aren't required to provide?

Reply:
> Could you comment out any config options here that aren't required to provide?

ok

Reply:
> Could you comment out any config options here that aren't required to provide?

Well, I found that retry_interval_ms is useless, I will delete it, but other configurations are necessary